tracer/test/test-applications/integrations/Samples.GoogleProtobuf/Samples.GoogleProtobuf.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">3.28.2</ApiVersion>
```

Comment:
Wondering if you tested with older versions as well?

Reply:
Hmm, I don't think I did. Do you think I should just run it once locally, or add test cases ?

Reply:
I was just thinking manually swapping it to an older version locally just to see that it works?

Reply:
wow turns out testing on older versions is super annoying because you have to use the protobuf generator (protoc) that matches the version of protobuf you are using.